db/schema.rb

```diff
@@ -192,14 +192,14 @@
     t.integer "hb"
     t.integer "hbrentshortfall"
     t.integer "property_relet"
-    t.datetime "mrcdate", precision: nil
+    t.datetime "mrcdate"
```

Comment:
not sure why these are being removed but it doesn't make a difference afaik, default precision either way